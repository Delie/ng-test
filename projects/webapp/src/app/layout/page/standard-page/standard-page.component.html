<app-header></app-header>
<div class="page-content">
  <ng-content></ng-content>
</div>
<app-footer></app-footer>
