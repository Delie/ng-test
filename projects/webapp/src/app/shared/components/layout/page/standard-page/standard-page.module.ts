import { NgModule } from '@angular/core';
import { StandardPageComponent } from './standard-page.component';
import { HeaderModule } from '../../header/header.module';
import { FooterModule } from '../../footer/footer.module';

@NgModule({
  declarations: [StandardPageComponent],
  imports: [HeaderModule, FooterModule],
  exports: [StandardPageComponent]
})
export class StandardPageModule {}
