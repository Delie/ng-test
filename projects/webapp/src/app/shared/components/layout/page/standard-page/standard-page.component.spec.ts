import { async } from '@angular/core/testing';
import { Shallow } from 'shallow-render';
import { StandardPageComponent } from './standard-page.component';
import { StandardPageModule } from './standard-page.module';

describe('StandardPageComponent', () => {
  let shallow: Shallow<StandardPageComponent>;
  beforeEach(async(() => {
    shallow = new Shallow(StandardPageComponent, StandardPageModule);
  }));

  it('should create the component', async () => {
    const { element } = await shallow.render();
    expect(element.nativeElement).toMatchSnapshot();
  });
});
