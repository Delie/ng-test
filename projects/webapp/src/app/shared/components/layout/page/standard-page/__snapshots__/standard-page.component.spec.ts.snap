// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StandardPageComponent should create the component 1`] = `
<app-standard-page>
  <app-header />
  <div
    class="page-content"
  />
  <app-footer />
</app-standard-page>
`;
